inc/search.class.php

```diff
@@ -4027,6 +4033,11 @@ static function addWhere($link, $nott, $itemtype, $ID, $searchtype, $val, $meta
 
          if (!empty($complexjoin)) {
             $table .= "_".$complexjoin;
+         } else {
```

Comment:
I guess the else is not necessary, but i preferred to be safe

Reply:
you should use a `elseif` and avoid unnecessary level